lib/assessment.rb

```diff
@@ -13,28 +13,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce { |sum, num| sum += num }
```

Comment:
Correct. This could also have been written even more shortly if you used the `&` trick:
`enum.reduce(&:+)`